crates/vfio-bindings/src/fam_wrappers.rs

```diff
@@ -89,6 +89,7 @@ mod tests {
         irq_set_vec[0].start = 0;
         irq_set_vec[0].count = event_fds.len() as u32;
 
+        // SAFETY: irq_set_vec is a valid flexible array constructed by us.
```

Comment:
nit: for tests you can ignore the safety requirements by writing the following line at the beginning of the tests module:
```
#![allow(clippy::undocumented_unsafe_blocks)]
```

Reply:
Okay. There is no need to change this now, right?